watchman.c

```diff
@@ -355,6 +381,13 @@ watchman_read(struct watchman_connection *conn, struct watchman_error *error)
     return result;
 }
 
+static json_t *
+watchman_read(struct watchman_connection *conn, struct watchman_error *error)
+{
+  struct timeval unused = {0};
```

Comment:
this value for select means "return immediately", rather than "block indefinitely" (a NULL timeout).  Is that what you mean?